### ISSUE TITLE
create_personalized_transcriptome.py transcript name processing from FASTA

### ISSUE THREAD

**Issue by averydavisbell** [SeyoungKimLab/Ornaments#4]
This is another picky "a certain format of name was expected but I'm using another one" note - in case helpful for making ornaments more flexible for different input types. (I've changed my copy of the code to work with my fasta naming format already.)

In running create_personalized_transcriptome.py, I noticed I was getting the same transcriptome fasta out as went in despite having lots of SNPs in the transcriptome VCF. Turns out this was because the transcriptome fasta sequence names are
`>transcript_id gene=gene_id`
e.g.,
`>2L52.1a.1 gene=WBGene00007063`
While the transcript names in the transcript VCF appropriately only have the transcript ID (e.g. 2L52.1a.1 )

This is because line 2204 was pulling transcript names based on splitting on a |:
`tname = tname.split('|')[0]`
Where my specific file needs:
`tname = tname.split(' ')[0]`

There's probably a clever re way to split on whatever character is in the fasta sequence name, but then you might get into other issues - maybe cleaner to require a fasta that just has the transcript ID in the sequence name? (though of course that's a bit annoying for users)

**Comment by azurillandfriend** [comment]
I agree with @averydavisbell comment above - probably the easiest option will be to add to the documentaion that the transcripts.fasta file needs to just have the >sequence_names without any spaces in. I got caught by this as mine had the gene name then space and a load of other stuff, and the convert_genomic_vcf_to_transcriptomic_vcf.py and create_personalized_transcriptome.py ran without error messages but the spaces in the sequence names caused it to skip all the seuqences and not find any shades. I then used cut -d" " -f1 Annotated_Transcripts.fasta > Transcripts.fasta 

It's also worth adding to the documentation to avoid the second pitfall I fell into which is that that it gets confused if the variants.vcf file at the start has format fields other than the GT, again running those two pythin scripts without producing an error message, but just failing to find any shades. The superfluous fields can be removed from the vcf using e.g. bcftools annotate -x "^FORMAT/GT"
